msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/ui/UIPermissionsManagerImpl.java

```diff
@@ -394,7 +394,9 @@ public boolean isMovePostings(DiscussionTopic topic, DiscussionForum forum)
       while (iter.hasNext())
       {
         DBMembershipItem item = (DBMembershipItem) iter.next();
-        if (item.getPermissionLevel().getMovePosting().booleanValue()
+        if ((item.getPermissionLevel().getMovePosting().booleanValue()
+            || item.getPermissionLevel().getReviseAny().booleanValue()
+            || item.getPermissionLevel().getReviseOwn().booleanValue())
```

Comment:
Why is being able to edit your own messages considered sufficient to be able to move a thread?